CONTRIBUTING.md

```diff
@@ -0,0 +1,135 @@
+# Contributing to FAAST
+
+Thanks for your interest in improving the project! This document provides a step-by-step guide for general contributions to FAAST.
+
+## Getting started
+
+There are there general ways you can contribute to this repo:
+
+- Proposing an enhancement or new feature
+- Reporting a bug or regression
+- Contributing changes to the source code
+
+For the first two, refer to the [GitHub Issues](https://github.com/faast-rt/lambdo/issues/new/choose) which guides you through the available options along with the needed information to collect.
+
+> To setup your development environment, please refer to the [README](./README.md).
+
+## Contributing changes
+
+_Prior to opening a pull request, it is recommended to open an issue first to ensure the maintainers can review intended changes. Exceptions to this rule include fixing non-functional source such as code comments, documentation or other supporting files._
+
+Proposing source code changes is done through [GitHub's standard pull request workflow](https://docs.github.com/en/get-started/quickstart/github-flow#create-a-pull-request).
+
+If your branch is a work-in-progress then please start by creating your pull requests as draft, by clicking the down-arrow next to the `Create pull request` button and instead selecting `Create draft pull request`.
+
+This will defer the automatic process of requesting a review from the FAAST team and significantly reduces noise until you are ready. Once you are happy, you can click the `Ready for review` button.
+
```

Comment:
We don't talk where the contributor should contribute. I think they should fork the repo and create PRs from their repo. Adding a small sentence about that seems relevant.